pkg/util/helm/helm_test.go

```diff
@@ -0,0 +1,337 @@
+package helm
```

Comment:
This file is too long; I had to scroll up and down a lot to see the whole picture. 
I suggest we split it into two/three files to make it more readable. For example, maybe test install/test delete can be in different files.

Reply:
The file had been split into two test files.